CONTRIBUTING.md

```diff
@@ -47,35 +47,35 @@ the diagrams. One of the diagrams is included below.
 The `master` branch contains stable, published code and is where release versions are tagged.
 Released versions will always be in the history of the `master` branch. The code on `master` uses
 published dependencies from `crates.io` and uses git dependencies only where absolutely necessary
-because relevant crates are not published on crates.
+likely because relevant crates.io does not yet host the relevant crates.
```

Comment:
```suggestion
likely because crates.io does not yet host the relevant crates.
```

Reply:
An alternative:

```suggestion
likely because the relevant crates are not published yet.
```
My thinking is that in the future perhaps some crates will be published to our own crates registry, crates.parity.io so perhaps it's even a good idea to start avoiding `crates.io`?

Reply:
I also considered "not yet published" when I was first writing this. Decided I didn't like it because putting the code on github for others to use also counts as publishing.